app/src/common/shared/com/igalia/wolvic/browser/PermissionDelegate.java

```diff
@@ -221,7 +221,8 @@ public WResult<Integer> onContentPermissionRequest(WSession aSession, ContentPer
             // https://hacks.mozilla.org/2019/02/firefox-66-to-block-automatically-playing-audible-video-and-audio/
             return WResult.fromValue(ContentPermission.VALUE_ALLOW);
         } else if(perm.permission == PERMISSION_AUTOPLAY_AUDIBLE) {
-            if (SettingsStore.getInstance(mContext).isAutoplayEnabled()) {
+            // allow autoplay when we launch Wolvic in immersive mode automatically
+            if (SettingsStore.getInstance(mContext).isAutoplayEnabled() || mWidgetManager.isLaunchImmersive()) {
```

Comment:
Not sure I understand why this is needed. Why do you want to force autoplay for immersive?

Reply:
Some experiences try to autoplay audio on launch.